src/Stratis.Bitcoin.Features.Dns/WhitelistManager.cs

```diff
@@ -92,6 +94,10 @@ public void RefreshWhitelist()
         {
             this.logger.LogTrace("()");
 
+            this.dnsPeerBlacklistThresholdInSeconds = this.dnsSettings.DnsPeerBlacklistThresholdInSeconds;
```

Comment:
Now we initialize the WhiteListManager's settings in the proper place.

Reply:
Yes.